test/conformance/runtime/header_test.go

```diff
@@ -124,6 +124,50 @@ func (c *checkIPList) String() string {
 	return c.expected
 }
 
+type checkForwardedHeader struct {
+	expected string
+}
+
+// token as defined in https://tools.ietf.org/html/rfc7230#section-3.2.6
+var tokenMatcher = regexp.MustCompile(`^[0-9a-zA-Z!#$%&'*+-.^_|~]+$`)
+
+// approximation of quoted-string as defined in https://tools.ietf.org/html/rfc7230#section-3.2.6
+var quotedStringMatcher = regexp.MustCompile(`^"[^"]+"$`)
```

Comment:
The way I read `A sender SHOULD NOT generate a quoted-pair in a quoted-string except
   where necessary to quote DQUOTE `, it feels that `\"` is valid inside quoted text. But, I do not think we have to be that precise. :)

Reply:
Right, hence it‘s an approximation 😬